src/BaGet/Entities/PostgreSqlContext.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Npgsql;
+
+namespace BaGet.Entities
+{
+    public class PostgreSqlContext : AbstractContext<PostgreSqlContext>
+    {
+        /// <summary>
+        /// The PostgreSql error code for when a unique constraint is violated.
```

Comment:
Please add a link to `https://www.postgresql.org/docs/9.6/errcodes-appendix.html` in this comment